### ISSUE TITLE
[npc] - "Appraiser" Sazsel Stickyfingers <Primal Trader>

### ISSUE THREAD

**Issue by forstix** [Valhalla-Gaming/Tracker#5051]
this npc <a href="http://www.wowhead.com/npc=84967/appraiser-sazsel-stickyfingers">link</a> is little overprice then the original (this is our only way to get elemental so..) 
<img src="http://imgur.com/57XuBcY" alt="Image">
ty.

**Comment by Aprophys** [comment]
yeah thats right i found someting but duno what garrison lvl it is but the prices are to high atm
http://www.wowhead.com/npc=84967/appraiser-sazsel-stickyfingers
